tools/Azure.Mcp.Tools.AzureManagedLustre/src/Commands/FileSystem/FileSystemImportJobCreateCommand.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Tools.AzureManagedLustre.Options;
+using Azure.Mcp.Tools.AzureManagedLustre.Options.FileSystem;
+using Azure.Mcp.Tools.AzureManagedLustre.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.AzureManagedLustre.Commands.FileSystem;
```

Comment:
Will there be many job-based tools? Might be worth putting this into a sub-namespace of `FileSystem` given `create` is too generic of a `Name`.